astropy/coordinates/transformations.py

```diff
@@ -38,6 +38,41 @@
            'FunctionTransformWithFiniteDifference', 'CompositeTransform']
 
 
+def frame_attrs_from_set(frame_set):
```

Comment:
Do you mean for this (and the next function) to be part of the public API?  I'm guessing not since it's not fully documented (and I'm OK with that), but if you do it needs to be in `__all__`.

Reply:
No, I think they should be private.